File: Kattis/Apaxiaaaaaaaaaaaans.cpp
//Link:-https://open.kattis.com/problems/apaxiaaans
#include <iostream>
using namespace std;
int main()
{
    string s;
    getline(std::cin,s);
    for(int i=0;i<s.length();i++)
    {
        if(s.length()==1){
         cout<<s[0];
         return 0;
        }
        if(s[i]!=s[i+1] && i!=s.length()-1)
         cout<<s[i];
        if(i==s.length()-1 && s[i]!=s[i-1])
         cout<<s[i];
    }
    return 0;
}

File: Kattis/Apaxiaaaaaaaaaaaans.java
//Link:-https://open.kattis.com/problems/apaxiaaans
import java.util.*;
public class Apaxiaaaaaaaaaaaans
{
    public static void main(String[] args)
    {
        Scanner in = new Scanner(System.in);
        String input = in.nextLine();
        String output = "";
        for (int i = 0; i < (input.length()); i++)
        {
           output += input.substring(i,i+1);
           int j;
           for(j =i+1;j<input.length();j++)
           {
               if(input.charAt(i)!=input.charAt(j))
                  break;
           }
           i=(j-1);
        }
        System.out.println(output);
    }
}
